db_stress_tool/expected_state.h

```diff
@@ -38,6 +38,16 @@ class ExpectedState {
   // Requires external locking covering all keys in `cf`.
   void ClearColumnFamily(int cf);
 
+  // Requires external locking
+  void SetPersistedSeqno(SequenceNumber seqno) {
+    persisted_seqno_->store(seqno, std::memory_order_relaxed);
```

Comment:
Do you think our use of `std::memory_order_relaxed` could end up biting us later down the road? I am not sure but maybe there is a scenario

Reply:
There will be lock externally so persisted_seqno_->store(seqno, std::memory_order_relaxed); won't get us the controversial std::memory_order_relaxed effect